tensorboard/webapp/app_routing/programmatical_navigation_types.ts

```diff
@@ -21,12 +21,14 @@ export const NAVIGATION_PROVIDER = new InjectionToken<NavigationLambda[]>(
 );
 
 export interface NavigateToExperiment {
+  replaceState?: boolean;
```

Comment:
Mind putting this either directly above or directly below resetNamespacedState?. It's about equal in importance. I would prefer to keep routeKind and routeParams at the top.

Reply:
Sure, I wasn't sure what the ordering convention was so I was going for alphabetical.